fluent-package/templates/package-scripts/fluent-package/deb/postinst

```diff
@@ -66,17 +66,23 @@ migration_from_v4_main_process() {
             echo "Migrating from /etc/<%= compat_service_name %>/plugin/ to /etc/<%= package_dir %>/plugin/..."
             mv -f /etc/<%= compat_service_name %>/plugin/* /etc/<%= package_dir %>/plugin/
         fi
+        create_link_for_config=n
+        if [ -f /etc/<%= compat_package_dir %>/<%= compat_service_name %>.conf -a ! -e /etc/<%= compat_package_dir %>/<%= service_name %>.conf ]; then
+            create_link_for_config=y
+        fi
         if [ -f /etc/<%= compat_package_dir %>/<%= compat_service_name %>.conf ]; then
             echo "Migrating from /etc/<%= compat_package_dir %>/<%= compat_service_name %>.conf to /etc/<%= package_dir %>/<%= compat_service_name %>.conf"
             cp -f /etc/<%= compat_package_dir %>/<%= compat_service_name %>.conf /etc/<%= package_dir %>/<%= compat_service_name %>.conf
-            echo "Refer previous configuration <%= compat_service_name %>.conf ..."
-            sed -i"" /etc/default/<%= service_name %> -e "/FLUENT_CONF/c FLUENT_CONF=/etc/<%= package_dir %>/<%= compat_service_name %>.conf"
-            for d in $(ls /etc/<%= compat_package_dir %>); do
-                if [ ! "$d" = "plugin" -a ! "$d" = "<%= compat_service_name %>.conf" ]; then
-                    # except managed under deb files must be migrated
-                    mv -f /etc/<%= compat_package_dir %>/$d /etc/<%= package_dir %>/
-                fi
-            done
+        fi
+        for d in $(ls /etc/<%= compat_package_dir %>); do
```

Comment:
With this for ... do, it seems that above `if [ -f /etc/<%= compat_package_dir %>/%<= compat_servie_name %>.conf ]; then ` block may be redundant.

Reply:
I'm overlooked the previous block condition, sorry.